instrumentation/README.md

```diff
@@ -1,181 +1,247 @@
-/usr/local/apache-tomcat/8.5.4/bin/tomcat-juli.jar \
--Dcatalina.base=/usr/local/apache-tomcat/8.5.4 \
--Dcatalina.home=/usr/local/apache-tomcat/8.5.4 \
--Djava.io.tmpdir=/usr/local/apache-tomcat/8.5.4/temp \
-org.apache.catalina.startup.Bootstrap start
-```
-
-produces a service name of:
-
-```
-org-apache-catalina-startup-bootstrap
-```
-
-## Disabling
-
-Short of uninstalling the package, there are a few safe ways of disabling auto instrumentation
-
-* Set `DISABLE_SPLUNK_AUTOINSTRUMENTATION` to any non-empty value other than false, FALSE, or 0
-* Set `JAVA_TOOL_OPTIONS` to some value that you want the JVM to pick up
-* Delete or move the config `instrumentation.conf` file
-
-## Files
-
-* /usr/lib/splunk-instrumentation/libsplunk.so
-* /usr/lib/splunk-instrumentation/instrumentation.conf
-* /usr/lib/splunk-instrumentation/splunk-otel-javaagent.jar
+# Splunk OpenTelemetry Zero Configuration Auto Instrumentation for Java
+
+**Splunk OpenTelemetry Zero Configuration Auto Instrumentation for Java** installs and enables the [Splunk
+OpenTelemetry Auto Instrumentation Java Agent](https://github.com/signalfx/splunk-otel-java) to automatically instrument
+Java applications running as systemd services on Linux, send the captured distributed traces to the locally running
+[Splunk OpenTelemetry Collector](https://docs.splunk.com/Observability/gdi/opentelemetry/opentelemetry.html), and then
+on to [Splunk APM](https://docs.splunk.com/Observability/apm/intro-to-apm.html).
+
+For Linux distributions that do ***not*** support Debian/RPM packages or Java applications ***not*** running as systemd
+services, see the [Instructions for app servers](
+https://docs.splunk.com/Observability/gdi/get-data-in/application/java/instrumentation/java-servers-instructions.html)
+for how to manually enable and configure the Java agent.
+
+> ***Note***: The `libsplunk.so` shared object file and related enablement/configuration processes are
+> ***deprecated***.  The `splunk-otel-auto-instrumentation` deb/rpm package is also ***deprecated*** and replaced by the
+> `splunk-otel-systemd-auto-instrumentation` deb/rpm package.
+> 
+> [Installation](#installation) of `splunk-otel-systemd-auto-instrumentation` will automatically:
+> - Uninstall the `splunk-otel-auto-instrumentation` deb/rpm package (if installed).
+> - Remove `/usr/lib/splunk-instrumentation/libsplunk.so` from `/etc/ld.so.preload` (if it exists).
+> - Enable the Java agent ***only*** for systemd services via the
+>   `JAVA_TOOL_OPTIONS-javaagent:/usr/lib/splunk-instrumentation/splunk-otel-javaagent.jar` systemd environment
+>   variable.
+>
+> For reference only, the documentation for `libsplunk.so` has been moved to [libsplunk.md](./libsplunk.md).
+
+## Prerequisites/Requirements
+
+- Check [Java agent compatibility and requirements](
+  https://docs.splunk.com/Observability/gdi/get-data-in/application/java/java-otel-requirements.html)
+- Debian or RPM based Linux distribution with the `systemd` service manager
```

Comment:
Can we stye `systemd` consistently across the README? We're using systemd in other instances, I'd go for `systemd` everywhere.